main.go

```diff
@@ -145,40 +111,62 @@ func main() {
 
 		webhookOpts.CARotateInterval, err = environment.LookupAsDuration("CA_ROTATE_INTERVAL")
 		if err != nil {
-			log.Error(err, "Failed retrieving ca rotate interval")
+			setupLog.Error(err, "Failed retrieving ca rotate interval")
 			os.Exit(1)
 		}
 
 		webhookOpts.CAOverlapInterval, err = environment.LookupAsDuration("CA_OVERLAP_INTERVAL")
 		if err != nil {
-			log.Error(err, "Failed retrieving ca overlap interval")
+			setupLog.Error(err, "Failed retrieving ca overlap interval")
 			os.Exit(1)
 		}
 
 		webhookOpts.CertRotateInterval, err = environment.LookupAsDuration("CERT_ROTATE_INTERVAL")
 		if err != nil {
-			log.Error(err, "Failed retrieving cert rotate interval")
+			setupLog.Error(err, "Failed retrieving cert rotate interval")
 			os.Exit(1)
 		}
 
 		if err := webhook.AddToManager(mgr, webhookOpts); err != nil {
-			log.Error(err, "Cannot initialize webhook")
+			setupLog.Error(err, "Cannot initialize webhook")
+			os.Exit(1)
+		}
+	} else if environment.IsOperator() {
+		// +kubebuilder:scaffold:builder
+		if err = (&controllers.NMStateReconciler{
+			Client: mgr.GetClient(),
+			Log:    ctrl.Log.WithName("controllers").WithName("NMState"),
+			Scheme: mgr.GetScheme(),
+		}).SetupWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create NMState controller", "controller", "NMState")
+			os.Exit(1)
+		}
+	} else if environment.IsHandler() {
+		// +kubebuilder:scaffold:builder
+		if err = (&controllers.NodeReconciler{
+			Client: mgr.GetClient(),
+			Log:    ctrl.Log.WithName("controllers").WithName("Node"),
+			Scheme: mgr.GetScheme(),
+		}).SetupWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create Node controller", "controller", "NMState")
 			os.Exit(1)
 		}
-	} else {
-		// Setup all Controllers
-		if err := controller.AddToManager(mgr); err != nil {
-			log.Error(err, "Cannot initialize controller")
+		// +kubebuilder:scaffold:builder
+		if err = (&controllers.NodeNetworkConfigurationPolicyReconciler{
+			Client: mgr.GetClient(),
+			Log:    ctrl.Log.WithName("controllers").WithName("NodeNetworkConfigurationPolicy"),
+			Scheme: mgr.GetScheme(),
+		}).SetupWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create NodeNetworkConfigurationPolicy controller", "controller", "NMState")
 			os.Exit(1)
 		}
 	}
 
 	setProfiler()
 
-	log.Info("Starting the Cmd.")
-	// Start the Cmd
-	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
-		log.Error(err, "Manager exited non-zero")
+	setupLog.Info("starting manager")
```

Comment:
Please keep upper/lower letters consistent while logging. All the logs before start with upper case

Reply:
Emitted code, I prefer to keep like the generated one.